bears/c_languages/CSecurityBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
```

Comment:
flawfinder is particularly hard to configure, it features only a regex and then only shows messages that match this regex, I can make a setting out of that I guess